tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/domain/Filer.java

```diff
@@ -40,100 +39,79 @@ public Filer setLanguagePlugin(LanguagePlugin languagePlugin) {
     public FileVisitResult decideOnDirectory(Path directory) {
         return FileVisitResult.CONTINUE;
     }
-
-    public List<String> prepareFile(List<String> file) {
-        return file;
-    }
-
-    List<CommentStyleFileFilter> getCommentStyleFileFilters() {
-        return new ImmutableList.Builder<CommentStyleFileFilter>()
-                .add(
-                        // Basic java
-                        new CommentStyleFileFilter(
-                                // JavaishCommentSyntax
-                                CommentSyntax.newBuilder()
-                                        .addSingleLineComment("\\/\\/")
-                                        .addMultiLineComment("\\/\\*+", "\\*+\\/")
-                                        .build()))
-                .add(
-                        new CommentStyleFileFilter(
-                                // XML comment syntax
-                                CommentSyntax.newBuilder()
-                                        .addMultiLineComment("/\\*", "\\*/")
-                                        .build()))
-                //TODO: add rest of the comment syntaxes
-                .build();
+    
+    public void visitFileExceptionWrapper(Path source, Path relativePath) {
+        try {
+            visitFile(source, relativePath);
+        } catch (IOException ex) {
+            throw new RuntimeException();
```

Comment:
`throw new RuntimeException(ex);` so that we will just wrap the original exception and it's message and stacktrace is visible.